homeassistant/components/withings/__init__.py

```diff
@@ -44,12 +44,6 @@
                 vol.Required(CONF_CLIENT_ID): vol.All(cv.string, vol.Length(min=1)),
                 vol.Required(CONF_CLIENT_SECRET): vol.All(cv.string, vol.Length(min=1)),
                 vol.Optional(const.CONF_USE_WEBHOOK, default=False): cv.boolean,
-                vol.Required(const.CONF_PROFILES): vol.All(
```

Comment:
Instead of removing this required configuration entry, can we mark it as deprecated in the scheme?

Reasoning: The integration will not break when a user still has it in the config and will throw a warning in the log instead.
You can still ignore it, in case the user has set it. It is just less of an annoying migration path for the end-user